src/database/models/users.js

```diff
@@ -1,52 +1,35 @@
 'use strict';
 module.exports = (sequelize, DataTypes) => {
   const users = sequelize.define('users', {
-    firstName: {
-      type: DataTypes.STRING,
-      AllowNull:{
-        args: false,
-        msg: "Please Enter the firstName"
-      }
-    },
-    lastName: {
-      type: DataTypes.STRING,
-      AllowNull: {
-        args: false,
-        msg: "Please Enter the lastName"
-      }
-    },
-    email: {
-      type:DataTypes.STRING,
-      AllowNull: {
-        args: false,
-        msg: "Please Enter your Email",
-       
-      }
-    },
-    password: {
-      type: DataTypes.STRING,
-      AllowNull: {
-        args: false,
-        msg: "Please enter your password"
-      },
-      },
-      isVerified: {
-        type:DataTypes.BOOLEAN,
-        AllowNull: {
-          args: true
-        }
-      },
-      signupType: {
-        type:DataTypes.BOOLEAN,
-        AllowNull: {
-          args: true
-        }
-      }
-    }
-  , {});
+    firstName: DataTypes.STRING,
+    lastName: DataTypes.STRING,
+    email: DataTypes.STRING,
+    password: DataTypes.STRING,
+    isVerified: DataTypes.STRING,
+    signupType: DataTypes.STRING
+  }, {});
   users.associate = function(models) {
     // associations can be defined here
   };
   return users;
 };
-
+/**
+ * @swagger
+ *  components:
+ *    schemas:
+ *      User:
+ *        type: object
+ *        required:
+ *          - email
+ *          - passoword
+ *        properties:
+ *          email:
+ *            type: string
+ *            format: email
+ *          name:
+ *            type: string
+ *        example:
+ *           email: fname.lname@andela.com
+ *           password: yourpassword
+ *
+ */
```

Comment:
I think no one is using this swagger doc schema.
We can get rid of it.

Reply:
okey vicky.